documentation/book/proc-removing-volumes-from-jbod-storage.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// assembly-storage.adoc
+
+[id='proc-removing-volumes-from-jbod-storage-{context}']
+= Removing volumes from JBOD storage
+
+This procedure describes how to remove volumes from Kafka cluster configured to use JBOD storage.
+It cannot be applied to Kafka clusters configured to use any other storage type.
+The JBOD storage always has to contain at least one volume.
+While removing volumes volumes, you should not change configuration of any other volumes or add new volumes at the same time.
+
+IMPORTANT: To avoid data loss, you have to move all partitions before removing the volumes.
+
+.Prerequisites
+
+* An {ProductPlatformName} cluster
+* A running Cluster Operator
+* A Kafka cluster with JBOD storage with two or more volumes
+
+.Procedure
+
+. Reassign all partitions from the disks which are you going to remove.
```

Comment:
I don't think we have a procedure which covers this. We have a procedure for cluster balancing, but that's not quite the same thing, and it doesn't cover moving partitions between data dirs on the same broker.

Reply:
I extended the chapter about reassigning partitions to cover the volumes as well. I also added subchapters to the storage modules to cover what the log dir path is.